headers/functions/overlay29.h

```diff
@@ -575,6 +577,7 @@ int TryAddTargetToAiTargetList(int current_num_targets, struct move_target_and_r
                                bool check_all_conditions);
 bool IsAiTargetEligible(struct move_target_and_range move_ai_range, struct entity* user,
                         struct entity* target, struct move* move, bool check_all_conditions);
+void TargetRegularAttack(struct entity* user, enum direction_id direction, bool check_petrified);
```

Comment:
`check_petrified` could be interpreted as "check the petrified status (in order to skip petrified enemies)" or "check petrified enemies", which are exact opposites. Can we rename this to something like `skip_petrified`?